packages/table/src/docs.md

```diff
@@ -277,6 +277,47 @@ Return a `EditableCell` component from the `renderCell` prop on a
 
 @interface IEditableCellProps
 
+@### Region
+
+A region is a rectangular group of cells in the table.
+
+Regions are typically used to describe boundaries for selections (via the
+`selectedRegions` prop) and/or custom overlays (via the `styledRegionGroups`
+prop). You may also wish to scroll directly to a region in the table via the
+[`Table.scrollToRegion`](#table-js.instance-methods) instance method.
+
+There are four different types of regions, each represented by a different
+`RegionCardinality`:
```

Comment:
how is this object used in code? you mention it here but never show examples that use it.

Reply:
maybe it would make more sense to introduce this enum _after_ explaining `IRegion`? be like, "ok now that you know all that, here's the shorthand we developed for it."